README.md

```diff
@@ -29,91 +66,28 @@ Two important things to do in order to properly handle deep links from Facebook
-
-4.) __Facebook deep links seem to not work?__
-
-Branch uses the Facebook App Links protocol to pass the deep links through to your app from Facebook. Funny enough, if you also have a Facebook app configured in the developer portal and you choose 'Deep link from feed', Facebook ignores it's own protocol. Make sure to *uncheck* this option in your Facebook app.
-
-5.) __Why do I not see any installs after I reinstall?__
-
-We do a lot of smart things to give you an accurate read on the number of installs you actually have. The most common one is associating the user with the actual hardware ID of the phone. If a user uninstalls the app, then reinstalls, we'll know it's the same person from before and just register the user as 'open' instead of an 'install.' To register an install on the same phone again, see FAQ #2 about debugging.
-
-6.) __Chrome seems to take me to Google Play all the time. Why?__
-
-Chrome is very picky about opening up the app directly. Chrome utilizes the intent system to try to open up the app, and fails back to the Play Store far too easily. Here are 3 things to verify:
-
-  1. Make sure the package name for the test build matches the package registered in the Branch settings and matches the package in Google play.
-
-  2. Make sure the URI scheme of the test build matches the Branch settings.
-
-  3. Verify that you've added the proper host 'open' in the Manifest - see [here](https://github.com/BranchMetrics/Branch-Android-SDK#register-an-activity-for-direct-deep-linking-optional-but-recommended)
-
-
-7.) __Why is the most recent version of Chrome (40+) is showing me a page not found error?__
-
-The Google Chrome team decided that it didn't want to try to open up the app if a user manually entered in a link into Chrome - see the history of this issue [here](https://code.google.com/p/chromium/issues/detail?id=331571). The scope of the bug applies to people who copy and paste a Branch link into Chrome, whereas anyone who clicks on a link in Chrome or something that opens the link in Chrome will properly redirect.
-
-We've filed an issue plus documented how you can file your own with the Chrome team to get this changed. Please see [this document](https://github.com/BranchMetrics/Branch-Android-SDK/blob/master/ChromeHelp.md) for more ways you can help!
+Have questions? Need troubleshooting assistance? See our [FAQs](https://dev.branch.io/references/android_sdk/#faq) for in depth answers.
```

Comment:
the dev.branch.io/references is actually pulled from this document, so after you delete it from here, this link will die. Perhaps we want to move this FAQ to another section at the bottom?